pkg/sql/sqlbase/multirowfetcher.go

```diff
@@ -36,9 +37,10 @@ import (
 // this to avoid using log.V in the hot path.
 const debugRowFetch = false
 
-type kvFetcher interface {
-	nextKV(ctx context.Context) (bool, roachpb.KeyValue, error)
-	getRangesInfo() []roachpb.RangeInfo
+// KVFetcher ...
```

Comment:
I'm sure you know about this already, but just a reminder to fix this doc. Could add a TODO(dan) BEFORE MERGE :)